src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -21,222 +22,222 @@
-    <Compile Include="System\Reflection\Metadata\PortablePdb\SequencePoint.cs" />
-    <Compile Include="System\Reflection\Metadata\PortablePdb\SequencePointCollection.cs" />
-    <Compile Include="System\Reflection\Metadata\PortablePdb\Tables.Debug.cs" />
-    <Compile Include="System\Reflection\Metadata\PEReaderExtensions.cs" />
-    <Compile Include="System\Reflection\Metadata\TypeSystem\PropertyDefinition.cs" />
-    <Compile Include="System\Reflection\Metadata\Signatures\SerializationTypeCode.cs" />
-    <Compile Include="System\Reflection\Metadata\Signatures\SignatureAttributes.cs" />
-    <Compile Include="System\Reflection\Metadata\Signatures\SignatureCallingConvention.cs" />
-    <Compile Include="System\Reflection\Metadata\Signatures\SignatureHeader.cs" />
-    <Compile Include="System\Reflection\Metadata\Signatures\SignatureKind.cs" />
-    <Compile Include="System\Reflection\Metadata\Signatures\SignatureTypeCode.cs" />
-    <Compile Include="System\Reflection\Metadata\TypeSystem\StandaloneSignature.cs" />
-    <Compile Include="System\Reflection\Metadata\TypeSystem\TypeDefinition.cs" />
-    <Compile Include="System\Reflection\Metadata\TypeSystem\TypeLayout.cs" />
-    <Compile Include="System\Reflection\Metadata\TypeSystem\TypeReference.cs" />
-    <Compile Include="System\Reflection\Metadata\TypeSystem\TypeSpecification.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\CoffHeader.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\CorFlags.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\CorHeader.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\DebugDirectory\CodeViewDebugDirectoryData.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\DebugDirectory\DebugDirectoryEntry.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\DebugDirectory\DebugDirectoryEntryType.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\DirectoryEntry.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\Machine.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEBinaryReader.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEFileFlags.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEHeader.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEHeaders.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEMemoryBlock.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEReader.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\PEStreamOptions.cs" />
-    <Compile Include="System\Reflection\PortableExecutable\SectionHeader.cs" />
-    <Compile Include="System\Reflection\Throw.cs" />
-    <Compile Include="System\Reflection\System.Reflection.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)Properties\InternalsVisibleTo.cs" />
```

Comment:
This is going to mess with the flow of adding files in VS. 

How about leaving these paths intact and doing something like this to redirect elsewhere:

``` xml
<_OriginalCompile Include="@(Compile)" />
<Compile Remove="@(_OriginalCompile)" />
<Compile Include="@(_OriginalCompile->'$(SystemReflectionMetadataPath)%(Identity)')" />
```